feature-libs/checkout/b2b/styles/components/steps/shipping-address/_index.scss

```diff
@@ -0,0 +1 @@
+@import './cost-center';
```

Comment:
Does it makes sense to rename `shipping-address` folder to `cost-center`?

Reply:
same case as above. Kept it for consistency, but this case was a bit different. Since this is under 'steps', we have either 4 steps or 5 steps (depending on credit card or account).

Since cost-center is part of shipping-address step, I left it as is with what it was in the old one, but I'll change it. Let me know with what I said if you want it back :d